share/spice/metar/metar.js

```diff
@@ -0,0 +1,29 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_metar = function(api_result) {
+        
+        if (!api_result || api_result.Error) {
+            return Spice.failed('metar');
+        }    
+
+        Spice.add({
+            id: "metar",
+            name: 'Weather',
+            data: api_result,
+            meta: {
+                sourceName: "Aviation Weather REST API",
+                sourceUrl: "http://avwx.rest"
+            },
+            normalize: function(item) {
+                return {
+                    title: item["Station"], 
+                    subtitle: item["Time"],
```

Comment:
I see there's some other information in the response, would any of it be useful to add to the subtitles?

If the subtitle is an array of objects you an produce a subtitle that looks like:

> Time: 311853Z | Dewpoint: 07 | Wind Speed: 09

Alternatively if use the Info template, these can be put into an info box, like you see for our Wikipedia results